site/gatsby-site/playwright/seeds/aiidprod/incidents.ts

```diff
@@ -87,6 +89,7 @@ const incidents: DBIncident[] = [
         },
         // this field is currently present in the database but not mapped to any graphql fueld
         // "created_at": 1407974400000
+        implicated_systems: ["amazon"],
```

Comment:
You must also add an entity `amazon` to the entities seeds list so the entity relationship check doesn't throw an error.